rapid_pro/flow_generation/FlowGraph.py

```diff
@@ -0,0 +1,136 @@
+import abc
+import uuid
+
+
+def generate_rapid_pro_uuid():
+    return str(uuid.uuid4())
+
+
+class DefinitionFile:
+    def __init__(self, flow_graphs):
+        self._flow_graphs = flow_graphs
+
+    def to_rapid_pro_dict(self):
+        return {
+            "version": "13",
+            "site": "https://textit.com",
```

Comment:
I will remove this "site" entry as it's not needed (I think it might be more of an 'origin' indicator).

I don't think anything else needs to be configured. expire_after_minutes is potentially configurable but we haven't needed that option to date, and I think the rest are necessary constants.